src/include/wiredtiger.in

```diff
@@ -1869,9 +1868,10 @@ struct __wt_session {
 	 * @config{commit_timestamp, set the commit timestamp for the current transaction.  The
 	 * supplied value must not be older than the first commit timestamp already set for the
 	 * current transaction\, if any.  The value must also not be older than the current oldest
-	 * and stable timestamps.  A commit timestamp is required for the commit of a prepared
-	 * transaction.  If the prepare timestamp has already been set\, the commit timestamp must
-	 * not be older than it.  See @ref timestamp_txn_api., a string; default empty.}
+	 * and stable timestamps.  For prepared transactions\, a commit timestamp is required for
+	 * commit\, must not be older than the prepare timestamp\, may be set only once\, and may
```

Comment:
Why are we saying `may be` here and all the error messages below? It is true that we don't allow those operations. Why can't we change it to `must`? Is there any scenario that we don't?

Reply:
I'm slightly confused because "may only be set once" means "setting it once is the only thing permitted"... at least to me. But if this fails for you it'll fail for other people, so it should be changed.

Does "can be set only once" and "cannot be set until after..." (and "cannot be set before" for the messages) work for you, or does it need to be "must be set only once" and "must not be set until after" (and "must not be set before")? The latter two are ok for me but the first seems odd in a way I can't quite put my finger on. Maybe "must not be set more than once"? Let me know what you prefer and I'll push changes...

Reply:
> I'm slightly confused because "may only be set once" means "setting it once is the only thing permitted"... at least to me.

I thought that I may be missing something while adding the comment. Thanks for the details. I am not a native speaker of English, which could be the reason.

> Does "can be set only once" and "cannot be set until after..." (and "cannot be set before" for the messages)

I am fine with any of the above messages.

Reply:
I changed the problematic one to "can" and the messages to "must".

There are lots of non-native speakers and they need to be able to read the docs. I never would have guessed this would have been a problem, so thanks :-)